code/modules/admin/verbs/listahelps.dm

```diff
@@ -0,0 +1,307 @@
+/datum/adminticket
+	var/ID = "" //ID of the ticket, very important as its used to find adminhelps.
+	var/user = "" //The user of the ahelp.
+	var/uckey //The saved ckey of the adminheloing user.
+	var/admin = "N/A" //The handling admin? Like come on.
+	var/msg = "" //The adminhelp message.
+	var/resolved = "No" //Is it resolved? Its much easier to have a "Yes" or a "No", as you can directly concat it into strings making life that much easier.
```

Comment:
I honestly have no idea what to put for defines. or how to properly write  out things for lazylist.  I wish I was actually trained, rather than the language reading = ok but writing = null hah.